### DISCUSSION TITLE
http instead of https in /ui

### DISCUSSION THREAD

**Discussion by rubenskuhl** [irrdnet/irrd#889] [Q&A]
Hi there. Many hrefs are generated the /ui code with http instead of https. When information is not being submitted, this is a lesser problem since the web server redirects any http request to https. But when submitting something, like passwords or objects, the page tries sending to http... which triggers an undesirable browser warning, and reflects the risks of sending that information in clear text over the network. 

Although mentioned for 2FA, url: "https://bgp.net.br" is configured so the server could know that https is in place. 

Is there anything I could for the time being ?  IRRd version is 4.4.2.

**Comment by mxsasha** [comment]
Is your webserver setting X-Forwarded-Proto? e.g. `proxy_set_header X-Forwarded-Proto $scheme;` [for nginx](https://irrd.readthedocs.io/en/stable/admins/deployment/#https-services-configuration). I am guessing the redirect might depend on that - which is not well documented if that's the issue.
